### PR TITLE
[Admin] Form validation error 

### PR DESCRIPTION
**Pull request by Tomanhez**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

**Random form screenshots:**

Screenshot 2019-07-03 at 14 51 19

Screenshot 2019-07-03 at 14 51 46

Screenshot 2019-07-03 at 14 58 07

### CODE REVIEW

File: src/Sylius/Bundle/AdminBundle/Resources/views/Crud/Create/_content.html.twig [hunk 1]
```diff
@@ -7,8 +7,10 @@
 <div class="ui segment">
     {{ form_start(form, {'action': path(configuration.vars.route.name|default(configuration.getRouteName('create')), configuration.vars.route.parameters|default({})), 'attr': {'class': 'ui loadable form', 'novalidate': 'novalidate'}}) }}
     {% if configuration.vars.templates.form is defined %}
+        {% include '@SyliusAdmin/form_validation_errors_checker.html.twig' %}
```
Comment (lchrusciel) [inline src/Sylius/Bundle/AdminBundle/Resources/views/Crud/Create/_content.html.twig]: Is there any reason, why we have to declare it twice? Won't it work properly, if we extract include before if statement?

**Comment by kulczy** [comment]
there is no need to create a new message template `form_validation_errors_checker.html.twig` we got one in the UiBundle `.../Macro/messages.html.twig`

File: src/Sylius/Bundle/AdminBundle/Resources/views/form_validation_errors_checker.html.twig [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{% import '@SyliusUi/Macro/messages.html.twig' as messages %}
+{% if form.vars.valid == false %}
```
Comment (GSadee) [inline src/Sylius/Bundle/AdminBundle/Resources/views/form_validation_errors_checker.html.twig]: Missing blank line before if

File: src/Sylius/Bundle/UiBundle/Resources/translations/messages.en.yml [hunk 3]
```diff
@@ -887,6 +887,7 @@ sylius:
         user_details: 'User details'
         username: 'Username'
         users_list_of_all_users_registered_in_store: 'Users <small>List of all users registered in store</small>'
+        invalid_form: 'This form contains errors'
```
Comment (GSadee) [inline src/Sylius/Bundle/UiBundle/Resources/translations/messages.en.yml]: Wrong order of keys

File: src/Sylius/Behat/Context/Ui/Admin/ManagingCustomerGroupsContext.php [hunk 4]
```diff
@@ -179,6 +179,22 @@ public function iShouldBeNotifiedThatCustomerGroupWithThisCodeAlreadyExists()
...
+     */
+    public function iShouldSeeMessageThatInformationAreIncompleteInCreateSection(): void
+    {
+        Assert::same($this->createPage->getMessageInvalidForm(), 'Error This form contains errors');
+    }
+
+    /**
+     * @Then I should see message that information are incomplete in edit section
+     */
+    public function iShouldSeeMessageThatInformationAreIncompleteInEditSection(): void
```
Comment (GSadee) [inline src/Sylius/Behat/Context/Ui/Admin/ManagingCustomerGroupsContext.php]: I would be for merging these two methods and use `CurrentPageResolverInterface` to resolve what page it is

**Comment by pamil** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "flash-message-validation-form"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Comment by pamil** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "flash-message-validation-form"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Comment by lchrusciel** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "flash-message-validation-form"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Comment by lchrusciel** [comment]
Thank you, Tomasz! :tada:

**Pull request merged**